src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -59,7 +52,7 @@ func NewClient(addr, appName string) *MobileClient {
 
 	hqfd := client.Configure(clientConfig.Client)
 
-	hqfdc := hqfd.DirectHttpClient()
+	hqfdc := hqfd.NewDirectDomainFronter()
```

Comment:
After that is fixed, we can probably remove `fronter` as a field in `MobileClient`, but that's not a pressing issue of course.